Makefile

```diff
@@ -29,9 +29,15 @@ generate:
 test:
 	go test -count=1 -cover -race -timeout 30s -short ./...
 
+.PHONY: download-csm-common
+download-csm-common:
+	# TODO point to main branch
```

Comment:
Remove the TODO and add the correct URI,

Reply:
Yes this will be updated after the dell/csm PR is merged